src/api/resolvers/qrCodeResolvers.js

```diff
@@ -0,0 +1,35 @@
+import getWeb3 from '../web3'
+export const defaults = {}
+
+const resolvers = {
+  Query: {
+    async scanQRCodeSupported() {
```

Comment:
```suggestion
    async scanQRCodeSupported() {
      return !!web3.currentProvider.scanQRCode
    }
```

Reply:
If I do this, how does the following query changes? 

```
export const QRSupportedQuery = gql`
  query scanQRCodeSupported {
    scanQRCodeSupported @client {
      supported
    }
  }
`
```

Reply:
```suggestion
export const QRSupportedQuery = gql`
  query scanQRCodeSupported {
    supported: scanQRCodeSupported @client
  }
```